tools/monorepo/check-changelogger-use.php

```diff
@@ -112,16 +115,15 @@ function debug() {
 	try {
 		$data = json_decode( file_get_contents( $base_path . '/' . $project_path . '/composer.json' ), true, 512, JSON_THROW_ON_ERROR );
 		if (
-			! isset( $data['require']['automattic/jetpack-changelogger'] ) &&
-			! isset( $data['require-dev']['automattic/jetpack-changelogger'] )
+			! isset( $data['extra']['changelogger'] )
 		) {
 			continue;
 		}
 	} catch ( Exception $e ) {
 		continue;
 	}
-	$data  = isset( $data['extra']['changelogger'] ) ? $data['extra']['changelogger'] : array();
-	$data += array(
+	$data                                   = isset( $data['extra']['changelogger'] ) ? $data['extra']['changelogger'] : array();
```

Comment:
Minor suggestion: if we're continuing above when `$data['extra']['changelogger']` isn't set, we shouldn't need this `isset` check here, correct?

```suggestion
	$data                                   = $data['extra']['changelogger'];
```